geoips/plugins/modules/output_checkers/image.py

```diff
@@ -0,0 +1,184 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test script for representative product comparisons."""
+
+import logging
+from os.path import splitext
+
+LOG = logging.getLogger(__name__)
+
+interface = "output_checkers"
+family = "standard"
+name = "image"
+
+
+def correct_type(fname):
+    """Determine if fname is an image file.
+
+    Parameters
+    ----------
+    fname : str
+        Name of file to check.
+
+    Returns
+    -------
+    bool
+        True if it is an image file, False otherwise.
+    """
+    if splitext(fname)[-1] in [".png", ".jpg", ".jpeg", ".jif"]:
+        return True
+    return False
+
+
+def outputs_match(plugin, output_product, compare_product, image_compare_threshold):
```

Comment:
We should probably add additional test scripts, and/or unit tests for image comparisons using different thresholds. Probably best to create an issue for this, but I'll wait for your opinions before doing so.

Reply:
We could just get started with unit tests and implement one here... I think integration test is fine (I added it to the abi.static Infrared test script), but we could set up fully fledged pytests before merging this.

Reply:
When doing unit tests for this, would it make sense to make a small set of 100x100 pixel images to test against? I'm okay with doing it here or in a new PR, whichever @evrose54 thinks is most appropriate. If it's done in a new PR we should get it done quickly after merging this one, though.

Reply:
I can make this my main priority after this is merged. I'd prefer to merge this first, that way I can access the output_checker_kwargs argument in the new PR and use it for unit testing. Seems different enough to separate that in a new PR.

Reply:
Sounds good. Create a new issue for the unit testing, then resolve this comment thread.

Reply:
Sounds good to me.  And I agree, create tiny images to use for the comparison unit tests.

Link to the new issue here, then resolve this conversation

Reply:
See https://github.com/NRLMMD-GEOIPS/geoips/issues/374